Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -203,22 +204,44 @@ try {
             else {
                 $finalUrl = $url
             }
-            # Check validity of URL
+
             if ($finalUrl -like 'http*://*') {
+                # Check validity of URL
                 try {
                     Invoke-WebRequest -Method Head -UseBasicParsing -Uri $finalUrl | Out-Null
```

Comment:
This check was added to fail early, if case the URL isn't accessible.
Now that apps are downloaded a couple of lines below, this check isn't necessary.